lib/JMAP/TestSuite/Entity/Email.pm

```diff
@@ -217,7 +213,7 @@ sub _import_batch {
   my $get_expect = $pkg->get_result;
 
   my $get_res = $account->tester->request([[
-    $get_method => { ids => [ $set_sentence->created_ids ] },
+    $get_method => { ids => [ map { $_->{id} } values %{ $sentence->arguments->{created} } ] },
```

Comment:
Hey, I just wrote the same diff in another repo!

Reply:
:)